src/v2/Apps/ArtistSeries/Components/ArtistSeriesHeader.tsx

```diff
@@ -138,7 +138,8 @@ const ArtistSeriesHeaderLarge: React.FC<ArtistSeriesHeaderProps> = props => {
                 justifyContent="flex-end"
                 alignItems="center"
               >
-                <HeaderImage src={resize(image.url, { height: 400 })} />
+                {/** The max width for the image is ~600px, so we need that */}
+                <HeaderImage src={resize(image.url, { width: 1200 })} />
```

Comment:
If we used srcSet with 1x/2x srcs these comments wouldn't be necessary. This is the reason I've stopped using the Image component in Palette 🤷